index.html

```diff
@@ -1,13 +1,344 @@
+              alt="men foto"
+              class="testimonials__photo"
+              >
+              <a href="https://www.facebook.com/leslie.mckinney.18" class="testimonials__link">
+              Leslie Mckinney
+              </a>
+              <h4>CEO</h4>
+              <p>Nice work, Certificates National Association of Realtors (USA)</p>
+            </div>
+
+            <div class="testimonials__people">
+              <img
+              src="./images/people/2.png"
+              alt="woman foto"
+              class="testimonials__photo"
+              >
+              <a href="https://www.facebook.com/scarlett.hawkins.7773" class="testimonials__link">Scarlett Hawkins</a>
+              <h4>Senior Vice President</h4>
+              <p>My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my
+                heart
+              </p>
+            </div>
+            <div class="testimonials__people">
+              <img
+                src="./images/people/3.png"
+                alt="men foto"
+                class="testimonials__photo"
+              >
+              <a href="https://www.facebook.com/max.henry.1217" class="testimonials__link">Max Henry</a>
+              <h4>Chief Design Officer</h4>
+              <p>When my lovely valley of steam rises and half-day sun is above an impermeable</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="section">
+        <h3 class="section__title" id="contact">Contact us</h3>
+        <div class="contact">
+
+          <div class="contact__item">
+            <div class="contact__info">
+              <div class="contact__about">
+                <h4>Phone</h4>
+                <a class="contact__link" href="tel:+1 234 5555-55-55">+1 234 5555-55-55</a>
+                <h4>Email</h4>
+                <a class="contact__link" href="mailto:hello@miami.com">hello@miami.com</a>
+              </div>
+              <div class="contact__about">
+                <h4>Address</h4>
+                <a class="contact__link" href="https://cutt.ly/Cym6sYM">
+                400 first ave. n.
+                suite 700
+                Minneapolis, MN 
+                55401
+
+                </a>
+              </div>
+            </div>
+
+            <form action="" class="form">
+
+              <input
+                class="form__input"
+                type="text"
+                placeholder="Name"
+                required
+              >
+              <input
+                class="form__input"
+                type="text"
+                placeholder="Phone"
+                id="phone_consultation"
+                required
+              >
+              <textarea
+                class="form__input"
+                name="message"
+                id="message"
+                rows="10"
+                placeholder="Message"
+              >
+              </textarea>
```

Comment:
```suggestion
              <textarea
                class="form__input"
                name="message"
                id="message"
                rows="10"
                placeholder="Message"
              ></textarea>
``` і тоді placeholder `Message` буде видно

Reply:
![mobile version](https://user-images.githubusercontent.com/32794096/81949590-aed19980-960b-11ea-8bd6-3d5c0e84958e.PNG)
тут можеш зробити все однією колонкою

Reply:
Дяк, змінив